composer.json

```diff
@@ -46,7 +35,9 @@
         "behat/mink-selenium-driver": "*",
         "behat/mink-sahi-driver": "*",
         "behat/mink-selenium2-driver": "*",
-        "ezsystems/behatbundle": "@dev"
+        "ezsystems/behatbundle": "@dev",
+        "sensio/generator-bundle": "~2.3",
+        "egulias/listeners-debug-command-bundle": "~1.9"
```

Comment:
`generator-bundle` and `listeners-debug-command-bundle` are _not_ dev dependencies. Dev deps means they're needed for running tests mainly. This is not the case here as they're needed for project development.